packages/generators/add-generator.js

```diff
@@ -14,6 +16,8 @@ const getPackageManager = require("@webpack-cli/utils/package-manager")
 const npmExists = require("@webpack-cli/utils/npm-exists");
 const entryQuestions = require("./utils/entry");
 
+const PROPS = Array.from(PROP_TYPES.keys());
```

Comment:
`.keys` already returns an array. Why did you use `Array.from`?

Reply:
When using a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set), it returns an Iterator instead of array. Hence, we have to use `Array.from` method.

Reply:
Aaah it was a `Set`! Fair enough, thanks!